webapp/channels/src/components/admin_console/license_settings/renew_license_card/renew_license_card.tsx

```diff
@@ -75,20 +74,22 @@ const RenewLicenseCard: React.FC<RenewLicenseCardProps> = ({license, totalUsers,
                 />
             </div>
             <div className='RenewLicenseCard__licensedUsersNum'>
-                <FormattedMarkdownMessage
-                    id='admin.license.renewalCard.licensedUsersNum'
-                    defaultMessage='**Licensed Users:** {licensedUsersNum}'
+                <FormattedMessage
```

Comment:
Instead of adding the `strong` tags to the translated text, could you actually remove the user count from these `FormattedMessage`s so that they looks like this?

```
<div>
    <strong>
        <FormattedMessage
            defaultMessage='Licensed Users: '
        />
    </strong>
    {license.Users}
</div>
```

That saves the translators some work because they won't need to work around the `<strong>` tags, and it'll make the translation code run a bit faster since it won't have to parse those either